ext/node/package_json.rs

```diff
@@ -199,11 +205,6 @@ impl PackageJson {
       scripts,
     };
 
-    CACHE.with(|cache| {
-      cache
-        .borrow_mut()
-        .insert(package_json.path.clone(), package_json.clone());
-    });
```

Comment:
Moved this up to the method that does loading from the file system. I previously created the other methods and accidentally left this here for loading from a string.